test/entity-store.js

```diff
@@ -70,21 +70,26 @@ suite('entity-store', function() {
 		});
 
 		test('get entity returns null if not in store', function() {
-			var entity = window.D2L.Siren.EntityStore.get('static-data/rubrics/organizations/text-only/199/groups/176/criteria/623/UNKNOWN1.json', '');
-			expect(entity).to.be.null;
+			return window.D2L.Siren.EntityStore
+				.get('static-data/rubrics/organizations/text-only/199/groups/176/criteria/623/UNKNOWN1.json', '')
+				.then(function(entity) {
+					expect(entity).to.be.null;
+				});
 		});
 
-		test('get entity returns entity sync', function(done) {
-			var request = window.D2L.Siren.EntityStore.fetch('static-data/rubrics/organizations/text-only/199/groups/176/criteria/623/0.json', '');
-			request.then(function() {
-				var entity = window.D2L.Siren.EntityStore.get('static-data/rubrics/organizations/text-only/199/groups/176/criteria/623/0.json', '');
-				var description = entity && entity.getSubEntityByClass('description').properties.html;
-				expect(description).to.equal('Proper use of grammar');
-				if (!done.done) {
-					done();
-					done.done = true;
-				}
-			});
+		test('get entity returns entity sync', function() {
```

Comment:
@mdgbayly `.get` returns a `Promise` after @jbydeley-d2l's changes. I didn't notice the distinction at the time. So 0.2.0+ might be breaking your stuff?

Reply:
@omsmith Yeh, thanks, i had previously noticed this, so i modified our usage of `get` to conditionally handle the return value as a promise based on whether it was `thenable`. We're currently still using the 0.1.0 version, but it should still work when BSI gets updated to 0.2.0.